resources/idlharness.js

```diff
@@ -1507,6 +1507,17 @@ IdlInterface.prototype.test = function()
     this.test_members();
 };
 
+// This supports both Constructor extended attributes and constructor
+// operations until all idl fragments have been updated.
+IdlInterface.prototype.constructors = function()
```

Comment:
Can you test this in resources/test/tests/unit/IdlInterface/constructors.html, so that it stays working during the transition and beyond? Otherwise it could be broken by an webidl2.js update in a way that we'd only notice by looking at the actual idlharness.js tests.